tinybucket.gemspec

```diff
@@ -13,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'http://hirakiuc.github.io/tinybucket/'
   spec.license       = 'MIT'
 
-  spec.files         = `git ls-files -z`.split("\x0")
+  spec.files         =  Rake::FileList['lib/**/*', 'LICENSE', 'Rakefile', 'Gemfile', 'tinybucket.gemspec', 'README.md', '.rubocop.yml']
```

Comment:
Include required files only to gem package. (exclude `/docs`, `/specs` and some files)